docs/environment/README.md

```diff
@@ -68,4 +68,5 @@ Environment Variable | Description
 `DSS_GCP_SERVICE_ACCOUNT_NAME` | "travis-test"
 `DSS_CHECKOUT_BUCKET_OBJECT_VIEWERS` | This list manages the GCP Users and serviceAccounts able to access direct URLs on the GS checkout bucket. Other GCP entities must use presigned urls, or checkout to an external GS bucket they have access to. 
 `AWS_SDK_LOAD_CONFIG` | Needed for Terraform to correctly use AWS assumed roles
-
+`ENABLE_AUDIT_LOGS` | Determines if an AWS cloud trial log group will be created for audit logs. 0 means it will not be created. 1 means it will be created. The default value is 0. 
```

Comment:
Should also include the warning from earlier: `Do not enable audit logs until after you have succesfully deployed the DSS at least once otherwise it will fail.`

Reply:
Fixed